src/modm/board/samd21_mini/board.hpp

```diff
@@ -86,10 +86,10 @@ using D12 = GpioA19;
 using D13 = GpioA17;
 
 // User LED (inverted, because connected to 3V3)
-using LedD13 = GpioInverted<GpioOutputA17>;
-using LedTx = GpioInverted<GpioOutputA27>;
-using LedRx = GpioInverted<GpioOutputB03>;
-using Leds = SoftwareGpioPort< LedTx, LedRx >;
+using LedD13 = GpioA17;
+using LedTx = GpioA27;
+using LedRx = GpioB03;
+using Leds = LedTx;
```

Comment:
```suggestion
// User LED (Rx and Tx LEDs inverted, because connected to 3V3)
using LedD13 = GpioA17;
using LedTx = GpioInverted<GpioOutputA27>;
using LedRx = GpioInverted<GpioOutputB03>;
using Leds = SoftwareGpioPort< LedTx, LedRx >;
```

![grafik](https://user-images.githubusercontent.com/2820734/112638306-66942200-8e3f-11eb-9b73-e3644afae47b.png)
https://cdn.sparkfun.com/datasheets/Dev/Arduino/Boards/sparkfun-samd21-mini-breakout-v10.pdf

Reply:
I know, but the problem is that the SAM GPIO implementation does not have `GpioInverted` or `SoftwareGpioPort` implemented…

Reply:
Ohhh :D

Maybe we (you) should create a issue about that and add the "help wanted" label?

Reply:
If I'm honest I would rather rework the [Targets support matrix](https://github.com/modm-io/modm#targets) to something that's generated from lbuild using the data we already accumulate in `generate_module_docs` (but perhaps not so slow). That would give a much more detailed look at what peripherals are supported.

Reply:
Also not sure if the "help wanted" is generally useful, so far I've not seen a contributor specifically fix things they are not personally motivated to do.

Reply:
I think this is also because we hardly have such "help-wanted" problems that are suitable for beginners.
We could also try to participate in events like [hacktoberfest](https://hacktoberfest.digitalocean.com/) to attract new contributors.